packages/react-storage/src/components/StorageBrowser/views/Controls/Table.tsx

```diff
@@ -383,7 +383,7 @@ export const LocationDetailViewTable = ({
                 return (
                   <TableDataText>
                     <Icon className={ICON_CLASS} variant="file" />
-                    {row.key}
+                    {row.key.slice(current?.prefix.length)}
```

Comment:
Fixes visual regression showing full `prefix` in `Name` column (same fix on line 415)